packages/datadog-instrumentations/src/jest.js

```diff
@@ -280,7 +280,7 @@ function jestAdapterWrapper (jestAdapter) {
         testSuiteFinishCh.publish({ status, errorMessage })
         if (environment.global.__coverage__) {
           const coverageFiles = extractCoverageInformation(environment.global.__coverage__, environment.rootDir)
-          if (coverageFiles.length) {
+          if (coverageFiles.length && environment.testEnvironmentOptions._ddTestCodeCoverageEnabled) {
```

Comment:
`environment` and `testEnvironmentOptions` are always initialized? Can't be `undefined` or `null`?
```suggestion
          if (coverageFiles.length && environment && environment.testEnvironmentOptions && environment.testEnvironmentOptions._ddTestCodeCoverageEnabled) {
```

Reply:
If we check for environment not being empty, we should do it before when the var gets initialized in line 265, as it's used the same way in multiple places.

Reply:
I don't think `environment` can be empty but it doesn't hurt to guard against it. I do it in https://github.com/DataDog/dd-trace-js/pull/2521/commits/5896c93cbda593c0de33057d556909f96b279a5e